aurora/grids.f90

```diff
@@ -127,19 +127,19 @@ subroutine time_steps(n, t, dtstart, itz, tinc, verbose, t_vals, i_save)
   REAL*8, INTENT(IN)             :: tinc(n)
   LOGICAL, INTENT(IN), OPTIONAL  :: verbose
 
-  REAL*8, INTENT(OUT)            :: t_vals(30000)   ! hard-coded max size! Seg-fault if exceeded
-  INTEGER, INTENT(OUT)           :: i_save(30000)   ! hard-coded max size! Seg-fault if exceeded
+  REAL*8, INTENT(OUT)            :: t_vals(200000)   ! hard-coded max size! Seg-fault if exceeded
```

Comment:
@antonellozito as discussed in person, I'd suggest that you revert this change. If you can figure out how to do real-time allocation, I'd be happy to include that as a solution!

Reply:
@fsciortino You are right, I forgot it as it is while I was testing. I'll revert this back.